src/Pages/3DPrintingConsole/RequestForm.js

```diff
@@ -54,7 +54,7 @@ function RequestForm(props) {
       color: 'danger',
       value: 'Delete',
       handleUpdates: e => {
-        props.handleDeleteData(item);
+        props.handleDelete(item, e);
```

Comment:
do you need to pass in the second parameter `e`? your function only takes one argument